FeedStoreChallenge/ObjectBoxFeedStore.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  ObjectBoxFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Riccardo Rossi - Home on 22/03/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import ObjectBox
+
+typealias StoreFeed = ObjectBoxFeedStore.StoreFeed
+typealias Cache = ObjectBoxFeedStore.Cache
+
+public class ObjectBoxFeedStore: FeedStore {
+	
+	// MARK: - Inner classes & enums
+	
+	class StoreFeed: Entity {
+		var id: Id = 0
+		var modelId: String = ""
+		var description: String?
+		var location: String?
+		var url: String = ""
+		var cache: ToOne<Cache> = nil
+		
+		init(modelId: String = "", description: String? = nil, location: String? = nil, url: String = "", cache: Cache? = nil) {
+			self.modelId = modelId
+			self.description = description
+			self.location = location
+			self.url = url
+			self.cache.target = cache
+		}
+	}
+	
+	class Cache: Entity {
+		var id: Id = 0
+		var timestamp: Double
+		// objectbox: backlink = "cache"
+		var feed: ToMany<StoreFeed> = nil
+		
+		init(timestamp: Double = 0.0) {
+			self.timestamp = timestamp
```

Comment:
Ideally, you should define an interface that prevents clients from creating the model with the wrong state. For example, `timestamp` shouldn't be *any* `Double` in the initializer. It should be a `Date`. 

And it shouldn't have a default value.

```suggestion
		init(timestamp: Date) {
			self.timestamp = timestamp.timeIntervalSinceReferenceDate
```

Reply:
Similar to here: https://github.com/essentialdevelopercom/ios-lead-essentials-feed-store-challenge/pull/122#discussion_r604378777